wgpu-core/src/command/bundle.rs

```diff
@@ -691,8 +695,8 @@ impl RenderBundleEncoder {
                     indexed: false,
                 } => {
                     let scope = PassErrorScope::Draw {
+                        kind: DrawKind::MultiDrawIndirect,
```

Comment:
```suggestion
                        kind: DrawKind::DrawIndirect,
```

since `count` is `None`.